ContactDetailsApi/V1/Boundary/Request/ContactDetailsRequestObject.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using ContactDetailsApi.V1.Domain;
+
+namespace ContactDetailsApi.V1.Boundary.Request
+{
+    public class ContactDetailsRequestObject
```

Comment:
You need to add a validator class/classes (and corresponding unit tests) for this request class.

Reply:
Validation is a separate ticket I think.

Reply:
Cool. We can do it there then.